packages/common/src/external/lecternClient.ts

```diff
@@ -0,0 +1,16 @@
+import { entities as dictionaryEntities, restClient as dictionaryRestClient } from '@overturebio-stack/lectern-client';
+
+const client = (schemaServiceUrl: string) => {
+	return {
+		async fetchDictionaryByVersion(name: string, version: string): Promise<dictionaryEntities.SchemasDictionary> {
+			try {
+				const newSchema = await dictionaryRestClient.fetchSchema(schemaServiceUrl, name, version);
+				return newSchema;
+			} catch (error) {
+				throw new Error(`Failed to fetch schema: ${error}`);
```

Comment:
not easy to see from this file, in Github, which other process would catch the error thrown here. would that be the error-handling middleware of an endpoint? or one of the utils?

Reply:
This error is `caught`  by the util/function that is calling this lecternClient, and then thrown to the error handler middleware who is mapping this error with the corresponding HTTP error code.